lib/project_types/script/messages/messages.rb

```diff
@@ -29,71 +30,72 @@ module Messages
           invalid_script_name_help: "Replace or remove unsupported characters. Valid characters "\
                                     "are numbers, letters, hyphens, or underscores.",
 
-          no_existing_apps_cause: "You don't have any apps.",
+          no_existing_apps_cause: "You don't have any apps in your Partner Dashboard.",
           no_existing_apps_help: "Create an app with {{command:shopify create}} or "\
                                  "visit https://partners.shopify.com/.",
 
           no_existing_orgs_cause: "You don't have any partner organizations.",
           no_existing_orgs_help: "Visit https://partners.shopify.com/ to create a partners account.",
 
-          no_existing_stores_cause: "You don't have any stores.",
+          no_existing_stores_cause: "You don't have any stores in your Partner Dashboard.",
           no_existing_stores_help: "Visit https://partners.shopify.com/%{organization_id}/stores/ to create one.",
 
-          project_exists_cause: "Directory with the same name as the script already exists.",
-          project_exists_help: "Use different script name and try again.",
+          project_exists_cause: "A directory with this same name already exists.",
+          project_exists_help: "Try again and enter a different name for the script.",
 
-          invalid_extension_cause: "Invalid extension point %s.",
+          invalid_extension_cause: "Invalid Script API %s.",
           invalid_extension_help: "Allowed values: %s.",
 
           invalid_language_cause: "Invalid language %s.",
           invalid_language_help: "Allowed values: %s.",
 
           invalid_config: "Can't change the configuration values because %1$s is missing or "\
-                          "it is not formatted properly.",
+                          "it isn't formatted properly.",
 
           missing_script_json_field_cause: "The script.json file is missing the required %s field.",
           missing_script_json_field_help: "Add the field and try again.",
 
           invalid_script_json_definition_cause: "The script.json file contains invalid JSON.",
           invalid_script_json_definition_help: "Fix the errors and try again.",
 
-          no_script_json_file_cause: "You are missing the required script.json file.",
+          no_script_json_file_cause: "The script.json file is missing.",
           no_script_json_file_help: "Create this file and try again.",
 
-          configuration_syntax_error_cause: "The script.json configuration schema is not formatted properly.",
+          configuration_syntax_error_cause: "The script.json is not formatted properly.",
           configuration_syntax_error_help: "Fix the errors and try again.",
 
-          configuration_missing_keys_error_cause: "The script.json configuration schema is missing required keys: "\
+          configuration_missing_keys_error_cause: "The script.json file is missing required keys: "\
```

Comment:
So this error is only raised when the schema entries are missing some key values. Ex.

```json
{
  ...
  "configuration": {
    "type": "single",
    "schema": [
      {
        "key": "paymentMethodName",
        // this entry is missing key-value pairs
      }
    ]
  }
}
```

I think we should figure out some terminology to make that more clear. Should we refer to them as `configuration schema entries`? `configuration entries`? I see you using `A configuration entry` below. @andrewhassan maybe you have an opinion?

Reply:
I think the error below is for the schema. This one is if there are some top-level keys missing. For example, they forgot to include the `type: single`. I think what Kate has is reasonable for that.

Reply:
I was just assuming based on the previous message, but it looks like you're right! Sounds good then